stripe/src/main/resources/templates/result.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html xmlns='http://www.w3.org/1999/xhtml' xmlns:th='http://www.thymeleaf.org'>
+    <head>
+        <title>Result</title>
+    </head>
+    <body>
+        <div th:if='${error}' th:text='${error}' style='color: red;'></div>
+        <div th:unless='${error}' style='color: green;'>Success!</div>
```

Comment:
Here's an idea: If you return the Charge object from the StripeService to the ChargeController, you could use some of the Charge response data to populate additional elements in the model for the result page. I don't know what's interesting to show, but you could pick something (maybe the charge id?) of interest.
What do you think?